DESCRIPTION

```diff
@@ -0,0 +1,15 @@
+Package: HUCAgg
+Type: Package
+Title: Functions to aggregate WBD HUC12s
+Version: 0.0
+Date: 2015-10-06
+Author: David Blodgett
+Maintainer: David Blodgett <dblodgett@usgs.gov>
+Depends:
+    maptools, sp, rgeos
```

Comment:
I will try to switch, I did use importFrom in my roxygen markup so it's not hard, but was having a hard time getting rgeos to be available without using depends if my memory serves me.